src/content/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react"
+interface IProps {
+  message: string
+}
+
+interface EventDetail {
```

Comment:
Is this interface really neccessary?

Reply:
Yes for Typescript. Since it's a custom event there's no real easy type to give it when oassy the events as params. Easy way is to use ``any`` but that doesn't pass lint check and this is the proper way to assure type safety